src/echo-app/build.gradle.kts

```diff
@@ -4,7 +4,7 @@ val npx = rootProject.extra.get("npx")
 
 task<Exec>("npmInstall") {
     commandLine(npm, "install")
-    dependsOn(":src:echo-common:npmInstall")
+    dependsOn(":src:ts-ratchet:npmInstall")
```

Comment:
I think the rest makes sense, why is this being changed?

Reply:
We should no longer need to depend on anything, updated